### PR TITLE
test: add push pull test without parameters

### PR DESCRIPTION
**Pull request by dsankouski**
fix: delete OS metadata file

[PT-2754]: get org name from environment variable

This allows easy org switch

[PT-2754]: add push pull tests

For this test, data prepared in temporary folder, from test-data folder.
Push test uploads that into the app, then pull functionality tested, comparing
pulled files with pushed ones.

### CODE REVIEW

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -12,10 +12,11 @@ jobs:
       - run: |
           yarn install
           yarn prepack
+          export ENV=test-sfdx-plugin
```
Comment (veloceapps) [inline .github/workflows/unit-tests.yml:15]: we test on studio-dev on purpose

File: test/commands/veloce/data/push.test.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { promisify } from 'node:util';
 import { exec as plainExec } from 'node:child_process';
 
 const exec = promisify(plainExec);
+const env = process.env.ENV || 'studio-dev';
```
Comment (veloceapps) [inline test/commands/veloce/data/push.test.ts:5]: no magical defaults, please

File: test/commands/veloce/fixref.test.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { exec as plainExec } from 'node:child_process';
 // import { writeFileSync } from 'node:fs'
 
 const exec = promisify(plainExec);
+const env = process.env.ENV || 'studio-dev';
```
Comment (veloceapps) [inline test/commands/veloce/fixref.test.ts:7]: no magical defaults

File: test/commands/veloce/source/pull.test.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { exec as plainExec } from 'node:child_process';
 // import { writeFileSync } from 'node:fs'
 
 const exec = promisify(plainExec);
+const env = process.env.ENV || '${env}';
```
Comment (veloceapps) [inline test/commands/veloce/source/pull.test.ts:7]: strange default

Comment (dsankouski) [reply test/commands/veloce/source/pull.test.ts]: removed

File: test/commands/veloce/fixref.test.ts (hunk 3 above)
Comment (dsankouski) [reply test/commands/veloce/fixref.test.ts]: removed in all places

File: test/commands/veloce/data/push.test.ts (hunk 2 above)
Comment (dsankouski) [reply test/commands/veloce/data/push.test.ts]: removed in all places

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (dsankouski) [reply .github/workflows/unit-tests.yml:15]: desicion changed to test on dedicated env

**Pull request merged**